Automattic/wp-calypso
# Disallow promoting posts in non-public sites/posts

#### Proposed Changes

* Disallow promoting posts in : 
* - Private Posts 
* - Protected posts (by password)
* - Private sites
* - Offlined sites (coming soon badge)

* Fix notice when you don't have available posts to promote
* 
![image](https://user-images.githubusercontent.com/43957544/194584804-300d5012-712c-4a06-bbfb-5f7ce333dfff.png)

#### Testing Instructions

- Go to Posts List, Page List, Stats page and see that you cannot promote content that is not published

#### Pre-merge Checklist



Related to #
